angular/projects/spark-angular/src/lib/components/sprk-masthead/sprk-masthead.stories.ts

```diff
@@ -20,6 +36,19 @@ export default {
       SprkMastheadAccordionComponent,
       SprkMastheadAccordionItemComponent,
       SprkMastheadSelectorComponent,
+      SprkMastheadBrandingDirective,
+      SprkMastheadLogoDirective,
+      SprkMastheadLinkDirective,
+      SprkMastheadNavItemDirective,
+      SprkMastheadNavItemsDirective,
+      SprkMastheadNavBarDirective,
+      SprkMastheadNavCollapsibleDirective,
+      SprkMastheadNavCollapsibleButtonComponent,
+      SprkTextDirective,
```

Comment:
We can get rid of anything that's not Masthead component in here since it's because it's in the import/export of the module

Reply:
@RVMendoza , we cannot because these are needed for documentation. If they aren't imported then we can't display them in the "subcomponents" and then users won't have information on the Input documentation. Anything that we want to show up in the docs needs to be imported and added in the subcomponents.